cmd/build.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func init() {
 	root.AddCommand(buildCmd)
-	buildCmd.Flags().StringP("builder", "b", "default", "Buildpack builder, either an as a an image name or a mapping name as defined in func.yaml")
+	buildCmd.Flags().StringP("builder", "b", "", "Buildpack builder, either an as a an image name or a mapping name as defined in func.yaml\nEmpty value implies default builder will be used.")
```

Comment:
This is an improvement, but I was thinking that the proper behavior would be for the CLI to just respect what's in func.yaml if no value is given.

Reply:
Actually it should.

>Empty value implies default builder will be used.

is not true, it will use what's in the config

Reply:
@lance I changed the help string

Reply:
Ah I see now, lgtm